src/midi/midi_fluidsynth.cpp

```diff
@@ -30,24 +30,24 @@
 
 #include "control.h"
 #include "cross.h"
+#include "fs_utils.h"
 
 MidiHandlerFluidsynth instance;
 
 static void init_fluid_dosbox_settings(Section_prop &secprop)
 {
 	constexpr auto when_idle = Property::Changeable::WhenIdle;
 
-	auto *str_prop = secprop.Add_string("soundfont", when_idle, "");
-
-	str_prop->Set_help("Path to a MIDI SoundFont file in .sf2 format.\n"
-	                   "An optional percentage will scale the SoundFont's levels.\n"
-	                   "For example: soundfont.sf2 50 will attenuate it by 50 percent.\n"
-	                   "The scaling percentage can range from 1 to 500.");
-
-	// TODO Handle storing soundfonts in specific directory and update
-	// the documentation; right now users need to specify full path or
-	// fall on undocumented FluidSynth internal algorithm for picking
-	// sf2 files.
+	auto *str_prop = secprop.Add_string("soundfont", when_idle, "default.sf2");
+	str_prop->Set_help(
+	        "SoundFont file in .sf2 format. Use either direct path to .sf2\n"
+	        "or name of a file in 'soundfonts' directory of the DOSBox\n"
+	        "configuration directory.\n"
+	        "Name 'default.sf2' will pick the default FluidSynth soundfont\n"
+	        "if it's installed on your OS.\n"
```

Comment:
"SoundFont file in .sf2 format. Use either the full path to an .sf2 or the name of an .sf2 inside the 'soundfonts' directory within your DOSBox configuration directory.\n"

Reply:
With new of the possible relative use, I've adjusted this a bit more.